compose.override.production.yml

```diff
@@ -27,6 +27,8 @@ services:
       - production_postgres_data_backups:/backups:z
     env_file:
       - ./.envs/.production/.postgres
+    deploy:
+      replicas: 0
```

Comment:
I'm not sure it matters? As it is, if you set the envvar to 1 (as I did in the .gitlab-ci.yml file) it overrides this anyways and runs the postgres container. It feels like having this also be a variable is asking for problems with two of them floating around.

Reply:
Yeah, if we we wanted to keep it as a variable, I think we'd just remove those lines entirely as it would just inherit appropriately. Either way though, I think think it's the same. Approving!